### PR TITLE
<RadioGroup/> - migrating to stylable

### PR DESCRIPTION
**Pull request by lironch**
<RadioGroup/> - migrating styles to stylable 

**Note:**
After the migration some e2e tests started to fail. it happen since they were using a dedicated focus-driver that was written to suit the scss implementation and not the stylable one. since these tests were in fact testing the HOC and not the component we decided to remove them and only test the visual look of the button when focused.

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5631/

File: src/RadioGroup/RadioGroup.st.css [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+}
+
+.vertical {
+  display: flex;
+  flex-direction: column;
+}
+
+.horizontal {
+  display: flex;
+}
```
Comment (EgoziE) [inline src/RadioGroup/RadioGroup.st.css:25]: In the spirit of "picnic"
can these rules be saved by using a box?

**Comment by lironch** [comment]
@EgoziE it will be great if you can review my last commit, it contains the removal of most of the e2e tests and adding the relevant visual tests to replace them.
https://github.com/wix/wix-style-react/pull/5631/commits/9fe707d99317d6781ae61428e90ae1c7487b54c7

File: src/RadioGroup/RadioButton/test/RadioButton.visual.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+const tests = [
+  {
+    describe: 'with focus',
+    its: [
+      {
+        it: 'should show focus indication',
+        componentDidMount: async () => {
+          const driver = createDriver();
+          await driver.focus();
```
Comment (EgoziE) [inline src/RadioGroup/RadioButton/test/RadioButton.visual.js:29]: Duplicated code, extract to a function

**Pull request merged**